src/style.css

```
@@ -1,3 +1,73 @@
+/* @import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap'); */
+
 body {
   margin: 0;
 }
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.header {
+  box-sizing: border-box;
+  padding: 0 50px;
+  background: #fff;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 0.5rem 1rem #b1b1b1;
+  min-width: 1024px;
+}
+
+.nav {
+  position: relative;
+  height: 60px;
+  display: flex;
+  align-items: center;
+}
+
+.nav__item {
+  margin: 0;
+  padding: 0;
+  padding-right: 20px;
+}
+
+.nav_link {
+  color: black;
+  text-decoration: none;
+  position: relative;
+  display: block;
+  padding: 23px 0;
+}
+
+.nav__item:last-child {
+  padding-right: 0;
+}
```

1. use `:not`
2. use margin to make an outer indent between siblings (not padding)
```suggestion
.nav__item:not(:last-child) {
  margin-right: 20px;
}
```